streams/src/test/java/org/apache/kafka/streams/integration/IQv2IntegrationTest.java

```diff
@@ -418,6 +418,8 @@ public String metricsScope() {
             })
         );
 
+        // do not use the harness streams
```

Comment:
Not sure what this comment means?

Reply:
I changed this, let me know if the new comment makes more sense or if you'd like it removed.

I was indicating that the KafkaStreams that the test harness set up for the test won't be used, and a different KafkaStreams was being substituted.